api.go

```diff
@@ -678,3 +678,34 @@ func PausePod(podID string) (VCPod, error) {
 func ResumePod(podID string) (VCPod, error) {
 	return togglePausePod(podID, false)
 }
+
+// ProcessListContainer is the virtcontainers entry point to list
+// processes running inside a container
+func ProcessListContainer(podID, containerID string, options ProcessListOptions) (ProcessList, error) {
```

Comment:
Since this changes the API, you'll need to also do the following:

- `interfaces.go`: Add this function to the `VC` interface).
- `implementation.go`: Add a simple wrapper like the one for `SetLogger()`).
-  `pkg/vcMock/types.go`: Again, add a function similar to `SetLoggerFunc`).
- `pkg/vcMock/mock.go`: Add a simple wrapper like the one for `SetLogger()`).
- `pkg/vcMock/mock_test.go`: Add a simple `TestProcessListContainer()`.

Reply:
done, thanks

Reply:
Wow that was fast! :smile: